config/environments/production.rb

```diff
@@ -14,24 +14,24 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
-  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
-  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
-  # `config/secrets.yml.key`.
-  config.read_encrypted_secrets = true
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
 
```

Comment:
will we need to implement the new way of managing secrets described here?
https://github.com/rails/rails/pull/30067

Reply:
> If we remove this will we need to implement the new way of managing secrets described here?

This app doesn't use encrypted secrets at all, so no.

But after reading that PR a couple of times, I now see that `/config/secrets.yml` should be deleted, as it's not useful anymore.